### DISCUSSION TITLE
Camera Hub

### DISCUSSION THREAD

**Discussion by workingwheel** [elgatosf/elgato#125] [Help] [answered]
I believe it would be highly beneficial to make the project open source and invite community contributions. I've been working on several exciting features like AI-powered teleprompter assistance, instant background changes using Stable Diffusion, and a deep fake puppet mode, all developed locally using Python. Integrating these directly into the software, which I primarily code in Python and C++, would enhance its capabilities significantly.

**Comment by JollyNabeel** [comment]
Great idea to make the project open source! Your AI teleprompter, background changes using [great camera](https://oldrollpro.com/) with Stable Diffusion, and deep fake puppet mode sound amazing. Integrating these with Python and C++ will be a huge enhancement. Excited to see this progress!

**Comment by GeekyEggo** [comment, marked as answer]
Hi @workingwheel, thanks for your feedback — it sounds like you're working on some really interesting projects! Whilst we don't have plans to open-source Camera Hub, there is the potential for more SDKs in the future that further enable integrating with our products. We recommend joining our [Marketplace Makers Discord](https://discord.gg/GehBUcu627) where you can meet like-minded Makers, discuss your projects, and keep up to date with the latest announcements from the team!
